helmfile.d/values/velero-sc.yaml.gotmpl

```diff
@@ -143,6 +143,8 @@ schedules:
       labelSelector:
         matchLabels:
           velero: backup
+      includedNamespaces:
+        - monitoring
```

Comment:
Can this be a separate commit as it looks like a separate fix from the exclusion of the capi namespaces?

Reply:
I guess since it excludes all namespaces but `monitoring`, not _just_ capi namespaces, I can have it as a separate commit